setup.py

```diff
@@ -0,0 +1,57 @@
+import os
+import setuptools
+
+# figure out the version
+about = {}
+here = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(here, "spccore", "__version__.py")) as f:
+    exec(f.read(), about)
+
+setuptools.setup(
+    # basic
+    name='spccore',
+    version=about["__version__"],
+    packages=setuptools.find_packages(exclude=["tests", "tests.*"]),
+
+    # requirements
+    python_requires='>=3.5.*',
+    install_requires=[
+        'requests>=2.21.0',
+        'keyring==12.0.2',
```

Comment:
consider upgrading to latest version if they are stable. maybe look into using other library.